model-optimizer/extensions/back/ChangeRangeOutputType.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import logging as log
+
+import numpy as np
+
+from mo.back.replacement import BackReplacementPattern
+from mo.graph.graph import Graph, Node
+from mo.middle.passes.convert_data_type import data_type_str_to_np
+from mo.utils.error import Error
+
+
+class ChangeRangeOutputType(BackReplacementPattern):
+    """
+    Change Range 'output_type' from fp64 to fp32 since not all plugins support fp64 data type.
```

Comment:
There is a similar transformation for Cast operation ChangeCastOutputType. Can we make a single transformation that sets correct 'output_type' if operation has it?

Reply:
thanks for your feedback. I also thought this but unfortunately `Cast` operation has different attribute name `dst_type` and has some other differences so if we make a single transformation this will not reduce lines of code. So it's better to keep as a separate transformation. @lazarevevgeny do you agree?

Reply:
We could have a map like `{'Convert': 'dst_type', 'Range', 'output_type'}`

Reply:
ok, will do this

Reply:
done